docs/api.md

```diff
@@ -362,7 +362,26 @@ particular user.
 
 - `client_id`: The id returned from client registration.
 - `client_secret`: The secret returned from client registration.
-- `code`: A string that was received from the [authorization][] endpoint.
+- `ttl`: (optional) Seconds that this access_token should be valid.
+  
+  Not including this field works so as to not break the v1 contract. It
+  is HIGHLY recommended that you use it. If passed, it or the maximum of
+  100 years, whichever is smaller, will be used, and passed back in the
+  response in the `expires_in` property.
+  
+  In v2, refresh tokens will be mandatory, even if a `ttl` property is
+  not passed. At that point, this property will only be a hint. To
+  prepare for that, not passing a `ttl` will return a `warning`
+  property in the response reminding of this eventual change.
+- `grant_type`: Either the string `authorization_code` or `refresh_token`.
+  - If `authorization_code`:
+    - `code`: A string that was received from the [authorization][] endpoint.
+  - If `refresh_token`:
+    - `refresh_token`: A string that received from the [token][]
+      endpoint specifically as a refresh token.
+    - `scope`: (optional) A subset of scopes provided to this
+      refresh_token originally, to receive an access_token with less
+      permissions.
```

Comment:
Woops, this was the first thing I wrote in this PR. It didn't reflect that changes that we'd always expire the token now. Updated the text.